bases/world_spi/dataset_config.yaml

```diff
@@ -0,0 +1,50 @@
+
+# Nome (slug) do conjunto no CKAN
+# Exemplos: br-ibge-populacao, br-tse-eleicoes
+name: world_spi
+
+# Título do conjunto, a ser exibido no mecanismo de busca.
+# Exemplo: População brasileira
+title: Social Progress Index
+
+# Qual organização disponibiliza os dados originais?
+# Opções: escolher dessa lista -> https://basedosdados.org/api/3/action/organization_list
+# Se a organização não estiver na lista acima ou o nome não estiver conforme o manual de estilo
+# criar ou renomear a organização em https://basedosdados.org/organization/
+# Exemplos: br-ibge, br-tse, br-rj-gov
+organization: world-spi
+
+# Descrição do conjunto
+notes: The Social Progress Index insights visualization gives you a holistic understanding of world progress from 2011-2022, as well as a granular set of information.
```

Comment:
hmm, "insights visualization" talvez tenha ficado um pouco confuso, tentaria deixar evidenciado a aplicabilidade e objetivo do dataset, como:  "The Social Progress Index data set (including twelve years of historical data) for research, educational, policymaking or other related purposes with attribution to the Social Progress Imperative."